### DISCUSSION TITLE
Is it possible 'tap to place' on Scene viewer?

### DISCUSSION THREAD

**Discussion by EugeneFitzher** [google/model-viewer#1880] [Q&A]
### Description

Please provide a detailed description of the bug, how to reproduce it, and the
expected behavior.  Always include a code snippet, screenshot, any errors
reported in the console, and the model to help us understand and fix the
problem.

#### Live Demo

https://glitch.com/edit/#!/model-viewer

### Browser Affected

- [. ] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Helios
- [ ] IE
- [ ] Safari

### OS

- [. ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### Versions

- model-viewer: v0.8.0
- three.js: r109

Hello, I have a question on starting AR mode.

Is 'Tap to place' function exists?
When I use AR on model viewer, the model spawned right after it detected stabilized place.
I want to place it where I want to put on.

Any one can help me, thanks.

**Comment by pushmatrix** [comment]
Hi @EugeneFitzher, model-viewer does not control the UX of AR mode. That is done by [Scene Viewer ](https://developers.google.com/ar/develop/java/scene-viewer)on Android, and AR QuickLook on iOS. All model-viewer does is launch the viewer on its respective platform.

**Comment by cdata** [comment]
@EugeneFitzher we are probably willing to support this kind of UX when doing AR via WebXR, but as @pushmatrix suggested, we can't really control the UX of Scene Viewer.
